sos/report/plugins/openstack_aodh.py

```diff
@@ -10,16 +10,15 @@
 # See the LICENSE file in the source distribution for further information.
 
 import os
-from sos.report.plugins import Plugin, RedHatPlugin
+from sos.report.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
 
 
-class OpenStackAodh(Plugin, RedHatPlugin):
+class OpenStackAodh(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin):
```

Comment:
If you're going to be adding distro specific subclasses, these tagging classes go in the subclasses, not the base class.

Reply:
Done.